app/upgrades/testnet/v3/constants.go

```diff
@@ -0,0 +1,8 @@
+package v3
+
+const (
+	UpgradeName = "tv3"
+	// TODO: DEFINE UPGRADE HEIGHT
```

Comment:
we'll have a new PR for this later?

Reply:
I think this is just a placeholder, will update. We'll do a manual migration

For ref see: https://hackmd.io/@fedekunze/cosmos-upgrade-guide